docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -21,6 +21,7 @@ vars:
   deployment_name: hcls-01
   region: us-central1
   zone: us-central1-c
+  bucket_force_destroy: false
```

Comment:
nit: I have a slight preference for `force_destroy _buckets` as it reads more like a question. But what you have is also fine so feel free to ignore.